api/DOMStringList.json

```diff
@@ -172,10 +172,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
I'm pretty sure this is a problem with http://mdn-bcd-collector.appspot.com/tests/api/DOMStringList. The `length` property was probably just on the instance before this. Can you re-test with an instance?

Reply:
I'll have to look into how to obtain an instance for this interface -- for now, I've manually updated this based upon the [IDL in Safari 5.1.7](https://trac.webkit.org/browser/webkit/releases/Apple/Safari%205.1.7/WebCore/dom/DOMStringList.idl).